registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/packet/validator/PacketValidateProcessor.java

```diff
@@ -154,6 +158,8 @@ public class PacketValidateProcessor {
 	private static final String PRE_REG_ID = "mosip.pre-registration.datasync.store";
 	private static final String VERSION = "1.0";
 
+	String registrationId = null;
```

Comment:
while resolving the conflicts you have copied to above ,this is the wrong place its should be within process

Reply:
done